wildfly-standalone-centos8/scripts/wildfly-setup.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+adddate() {
+    while IFS= read -r line; do
+        printf '%s %s\n' "$(date "+%Y-%m-%d %H:%M:%S")" "$line";
+    done
+}
+
+echo "WILDFLY 18.0.1.Final Standalone Intallation Start..." | adddate >> wildfly.install.log
+/bin/date +%H:%M:%S  >> wildfly.install.log
+
+WILDFLY_USER=$1
+WILDFLY_PASSWORD=$2
+IP_ADDR=$(hostname -I)
+
+echo "WILDFLY_USER: " ${WILDFLY_USER} | adddate >> wildfly.install.log
+
+echo "WILDFLY Downloading..." | adddate >> wildfly.install.log
+echo "yum install -y git unzip java" | adddate >> wildfly.install.log
+yum install -y git unzip java | adddate >> wildfly.install.log 2>&1
+echo "yum -y install wget" | adddate >> wildfly.install.log
+yum -y install wget | adddate >> wildfly.install.log 2>&1
+WILDFLY_RELEASE="18.0.1"
+echo "wget https://download.jboss.org/wildfly/$WILDFLY_RELEASE.Final/wildfly-$WILDFLY_RELEASE.Final.tar.gz" | adddate >> wildfly.install.log
+wget https://download.jboss.org/wildfly/$WILDFLY_RELEASE.Final/wildfly-$WILDFLY_RELEASE.Final.tar.gz >> wildfly.install.log 2>&1
+flag=$?; if [ $flag != 0 ] ; then echo  "ERROR! Downloading WildFly Failed" | adddate >> wildfly.install.log; exit $flag;  fi
+echo "tar xvf wildfly-$WILDFLY_RELEASE.Final.tar.gz" | adddate >> wildfly.install.log
+tar xvf wildfly-$WILDFLY_RELEASE.Final.tar.gz | adddate >> wildfly.install.log 2>&1
+
+echo "echo "AllowTcpForwarding no" >> /etc/ssh/sshd_config" | adddate >> wildfly.install.log
+echo "AllowTcpForwarding no" >> /etc/ssh/sshd_config | adddate >> wildfly.install.log 2>&1
+echo "systemctl restart sshd" | adddate >> wildfly.install.log
+systemctl restart sshd | adddate >> wildfly.install.log 2>&1
+
+echo "Sample app deploy..." | adddate >> wildfly.install.log
+echo "git clone https://github.com/Suraj2093/dukes.git" | adddate >> wildfly.install.log

```

Comment:
you cannot pull from a personal repo in a QuickStart sample

Reply:
Moved this Sample App under the scripts folder and made the corresponding changes in the script